### ISSUE TITLE
[fvt]xcatprobe osdeploy -n node's result  -r 4h could not same with -r 40h

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1719]
env:sles11.4
build:
c910f02c08p08:~ # lsxcatd -v
Version 2.12.2 (git commit 9298b5ff211500e22398abd7c35b3fe28038feff, built Tue Aug 16 06:16:00 EDT 2016)
c910f02c08p08:~ # 
How to reproduce:
```
c910f02c08p08:~ # xcatprobe osdeploy -n c910f02c08p10 -r 4h1m 
All pre_defined nodes are valid                                                                                   [ OK ]
The installation network interface is eth0                                                                        [INFO]
Start to search logs after 'Wed Aug 17 00:15:05 2016', please waiting for a while.............

=============================================================
=  The summary of os provision:
=============================================================

[c910f02c08p10] deployment completed                                                                              [ OK ]
c910f02c08p08:~ # xcatprobe osdeploy -n c910f02c08p10 -r 40h1m
All pre_defined nodes are valid                                                                                   [ OK ]
The installation network interface is eth0                                                                        [INFO]
Start to search logs after 'Mon Aug 15 12:15:14 2016', please waiting for a while.............

=============================================================
=  The summary of os provision:
=============================================================

[c910f02c08p10] deployment failed, pass run_postscript stage, something wrong during server_reboot stage          [FAIL]
c910f02c08p08:~ # lsxcatd -v
Version 2.12.2 (git commit 9298b5ff211500e22398abd7c35b3fe28038feff, built Tue Aug 16 06:16:00 EDT 2016)
c910f02c08p08:~ # 
------------------------------> In fact c910f02c08p10 has been provisioned successfully
```

**Comment by hu-weihua** [comment]
@junxiawang , the process during  40 hours is different with the process during 4 hours in your environment. the root reason is there are processes except provision happened during 40 hours, ``osdeploy`` can't handle this complicated scenario very well now. we need to enhance the code logic.
but it seems can't be done in 2.12.2. so I move this issue to next release.

**Comment by hu-weihua** [comment]
Fixed by pull request #1866
